cmd/gokr-packer/packer.go

```diff
@@ -486,6 +486,10 @@ func overwriteFile(filename string, root *fileInfo, partuuid uint32, usePartuuid
 		return 0, 0, err
 	}
 
+	fmt.Printf("If your applications need to store persistent data, create a file system using e.g.:\n")
+	fmt.Printf("\t/sbin/mkfs.ext4 %s -F -E offset=%v\n", *overwrite, 8192*512+1100*MB)
```

Comment:
Is the /sbin/ prefix required? Is sudo not required? I would have expected “sudo mkfs.ext4”

Reply:
For most systems* /sbin isn't in the path for normal users (just root) and that's where the mkfs.ext4 tool lives, so yes.

If you were not running gokr-packer under sudo to write to the file/device in question then you don't need sudo to
write the filesystem onto it. 

My use-case was writing image files for testing in qemu and all of that under my own user account with no sudoing..

*citation needed

Reply:
Ah, thanks.